src/docs/development/platform-integration/add-to-app-android/add-flutter-fragment.md

```diff
@@ -0,0 +1,311 @@
+cannot be used for Android platform animations, nor can it be interleaved in the
+middle of an Android `View` hierarchy. If either of these use-cases are
+requirements for your app, you need to use `TextureView` instead of
+`SurfaceView`. Select a `TextureView` by building a `FlutterFragment` with a
+`texture` `RenderMode`:
+
+```java
+FlutterFragment flutterFragment = new FlutterFragment.Builder()
+    .renderMode(FlutterView.RenderMode.texture)
+    .build();
+```
+
+Using the configuration above, the resulting `FlutterFragment` renders its UI to
+a `TextureView`.
+
+## Display a `FlutterFragment` with transparency
+
+By default, `FlutterFragment` renders with an opaque background, using a
+`SurfaceView` (see the section above titled "Control `FlutterFragment`'s render
+mode"). That background is black for any pixels that are not painted by Flutter.
+Rendering with an opaque background is the preferred rendering mode for
+performance reasons. Flutter rendering with transparency on Android negatively
+impacts performance. However, there are many designs that require transparent
+pixels in the Flutter experience that show through to the underlying Android UI.
+For this reason, Flutter supports translucency in a `FlutterFragment`.
+
+{{site.alert.note}} 
+  Both `SurfaceView` and `TextureView` support transparency. However, when a
+  `SurfaceView` is instructed to render with transparency, it positions itself
+  at a higher z-index than all other Android `View`s, which means it appears
+  above all other `View`s. This is a limitation of `SurfaceView`, itself. If it
+  is acceptable to render your Flutter experience on top of all other content,
+  then `FlutterFragment`'s default `RenderMode` of `surface` is the `RenderMode`
+  that you should use. However, if you need to display Android `View`s both
+  below and above your Flutter experience, then you must specificy a
+  `RenderMode` of `texture`. See the section above titled "Control
+  `FlutterFragment`'s render mode" for information on controlling the
+  `RenderMode`. 
+{{site.alert.end}}
+
+To enable transparency for a `FlutterFragment`, build it with the following
+configuration:
+
+```java
+FlutterFragment flutterFragment = new FlutterFragment.Builder()
+    .transparencyMode(FlutterView.TransparencyMode.transparent)
+    .build();
+```
+
+## The relationships beween `FlutterFragment` and its `Activity`
+
+Some apps choose to use `Fragment`s as entire Android screens. In these apps, it
+would be reasonable for a `Fragment` to control system chrome like Android's
+status bar, navigation bar, and orientation.
+
+TODO: insert illustration of system chrome
+
+In other apps, `Fragment`s are used to represent only a portion of a UI. A
+`FlutterFragment` might be used to implement the inside of a drawer, or a video
+player, or a single card. In these situations, it would be inappropriate for the
+`FlutterFragment` to affect Android's system chrome because there are other UI
```

Comment:
Not sure what you mean by “Android’s system chrome”? Is this a typo?

Reply:
System chrome refers to things like the status bar, navigation bar, etc. Any semi-permanent area of the UI that is contributed by the OS instead of your app. The term system chrome is used within Android APIs, themselves.